iohub/ngff/nodes.py

```diff
@@ -969,19 +969,7 @@ def scale(self) -> list[float]:
         Helper function for scale transform metadata of
         highest resolution scale.
         """
-        scale = [1] * self.data.ndim
-        transforms = (
-            self.metadata.multiscales[0].datasets[0].coordinate_transformations
-        )
-        for trans in transforms:
-            if trans.type == "scale":
-                if len(trans.scale) != len(scale):
-                    raise RuntimeError(
-                        f"Length of scale transformation {len(trans.scale)} "
-                        f"does not match data dimension {len(scale)}."
-                    )
-                scale = [s1 * s2 for s1, s2 in zip(scale, trans.scale)]
-        return scale
+        return self.get_effective_scale("*").scale
```

Comment:
This used to be the scale of the highest resolution, so it should be the first array instead of `"*"`.

Reply:
I think this is why tests are failing for https://github.com/czbiohub-sf/iohub/pull/264/commits/6fc5ad6616eeccc49d14dd84c60d616429abc9e3.